css/_christmas.scss

```diff
@@ -0,0 +1,11 @@
+$color-bckg: #F5F4D7;
+$color-gray: #951F2B;
+
+.page-header__logo {
+    display: flex;
+}
+
+.page-header__logo::after {
+    content: url(https://i.ibb.co/WGbjjYX/tree.png);
```

Comment:
content sluzy tylko i wylacznie do wstawiania tekstu, trzeba go bylo zostawic pustym a nastepnie sprawic aby caly after byl elementem blokowym i wstawic mu choinke jako tło

podeslalem rozwiazanie to sobie tam zerknij